lib/easy.c

```diff
@@ -84,11 +84,25 @@
 #include "curl_printf.h"
 #include "curl_memory.h"
 #include "memdebug.h"
+#include "easy_lock.h"
 
 /* true globals -- for curl_global_init() and curl_global_cleanup() */
 static unsigned int  initialized;
```

Comment:
I think this is going to have to be marked volatile to ensure that the right value is used in all threads.

Reply:
The pthread_mutex and the SRWLock use a memory barrier, so volatile here is not needed.